packages/imt.sol/contracts/package.json

```diff
@@ -1,7 +1,7 @@
 {
-    "name": "@zk-kit/incremental-merkle-tree.sol",
-    "version": "1.7.0",
-    "description": "Incremental Merkle tree Solidity libraries.",
+    "name": "@zk-kit/imt.sol",
+    "version": "2.0.0-beta",
```

Comment:
Do we wanna keep the beta right now?

Reply:
Hey, yes! Until we test it with Semaphore v4 and have a audit 👍🏽